<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateManager</class>
 <widget class="QWidget" name="UpdateManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UpdateManager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="page_options">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButtonCheckUpdates">
         <property name="text">
          <string>Check for updates</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/check.png</normaloff>:/check.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="description">
          <string>Check online if %1 has a new vesion.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButtonStart">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/start.png</normaloff>:/start.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="description">
          <string>Start %1 with out checking for a new version.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxDontShow">
         <property name="text">
          <string>Don't check for updates when %1 starts.</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>commandLinkButtonCheckUpdates</zorder>
      <zorder>commandLinkButtonStart</zorder>
      <zorder>verticalSpacer</zorder>
      <zorder>checkBoxDontShow</zorder>
     </widget>
     <widget class="QWidget" name="page_progress">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButton_cancelProgress">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/close.png</normaloff>:/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_results">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0,0,0">
       <item>
        <widget class="QLabel" name="label_downloadPage">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Leave this empty</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButtonDownload">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/download.png</normaloff>:/download.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="description">
          <string>Download the new %1 vesion in your browser.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButtonStart2">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/start.png</normaloff>:/start.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="description">
          <string>Start %1.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButtonExit">
         <property name="text">
          <string>Exit</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/close.png</normaloff>:/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="description">
          <string>Exit %1.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_error">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Error:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_error">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Leave this empty</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButtonStart3">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/start.png</normaloff>:/start.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="description">
          <string>Start %1.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="commandLinkButtonExit2">
         <property name="text">
          <string>Exit</string>
         </property>
         <property name="icon">
          <iconset resource="rc/resources.qrc">
           <normaloff>:/close.png</normaloff>:/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="description">
          <string>Exit %1.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
